engines/director/detection_tables.h

```diff
@@ -7785,6 +7786,9 @@ static const DirectorGameDescription gameDescriptions[] = {
 
 	WINGAME1("thesims", "", "maxis.exe", "d62438566e44826960fc16c5c23dbe43", 1915533, 650),
 
+	// Tivola demo/sampler disc, Summer 2000, released in Germany
+	WINGAME1_l("tivolasummer2000", "Sampler", "START32.EXE", "1b8d78ddca650041b8997cac7af3184b", 1675134, Common::DE_DEU, 650),
```

Comment:
For Windows titles, please use tail md5 checksum. The beginning of the file is always the same.